src/ctia/http/routes/common.clj

```diff
@@ -32,7 +38,10 @@
    (s/optional-key :tlp) s/Str})
 
 (s/defschema SourcableEntityFilterParams
-  {(s/optional-key :source) s/Str})
+  {(s/optional-key :source) s/Str
+
+   (s/optional-key :query_mode)
+   (describe FullTextQueryMode "Elasticsearch Fulltext Query Mode. Defaults to query_string")})
```

Comment:
"Sourcable" is related to fields that enable to identify the source of the identity, not to search it.
There is only source for now, but I still hope that one day we will get closer to the provenance approach of the Semantic Web 😉

Reply:
Hmm. I think I misunderstood your comment here: https://github.com/threatgrid/ctia/pull/1109#discussion_r625897294

Reply:
Like `SourcableEntityFilterParams` you would have `SearchableEntityParams`